redaxo/src/core/lib/clang/clang.php

```diff
@@ -82,6 +82,7 @@ public static function getStartId()
         foreach (self::getAll() as $id => $clang) {
             return $id;
         }
+        throw new rex_should_never_happen_exception();
```

Comment:
Ich würde eher direkt mit `LogicException` arbeiten und in der Regel auch mit aussagekräftigerer Message, selbst wenn es eigentlich eben gerade nicht vorkommen soll.

LogicException hat zum Beispiel den Vorteil, dass PhpStorm die nicht als `@throws`-Tag notieren will.

Also irgendwie sowas in der Art:

```suggestion
        throw new LogicException('No clang found.');
```

Reply:
done